arrows/klv/klv_read_write.h

```diff
@@ -207,6 +207,40 @@ KWIVER_ALGO_KLV_EXPORT
 size_t
 klv_ber_oid_length( T value );
 
+// ---------------------------------------------------------------------------
+/// Read an IEEE-754 floating-point number from a sequence of bytes
+/// (big-endian).
+///
+/// \param[in,out] data Iterator to sequence of \c uint8_t. Set to end of read
+/// bytes on success, left as is on error.
```

Comment:
Given the amount of code in this file which uses the non-indenting style, maybe it would be best to re-do the style for all functions all at once in a separate PR, once they're all merged in? I think all code that's going to end up in this file is already in a PR.

Reply:
Yeah, this is hardly world-ending 🙂.